Provide the example scripts no matter whether ruby is available in the 
build environment or not.


--- a/klinkstatus/src/plugins/scripting/scripts/CMakeLists.txt
+++ b/klinkstatus/src/plugins/scripting/scripts/CMakeLists.txt
@@ -5,9 +5,7 @@ install(FILES
 macro_optional_find_package(RUBY)
 macro_log_feature(RUBY_FOUND "Ruby" "An Interpreted object-oriented scripting language" "http://www.ruby-lang.org" FALSE "" "For KLinkStatus example ruby scripts")
 
-if(RUBY_FOUND)
     
-    include_directories(${RUBY_INCLUDE_PATH})
 
     install(FILES 
         examples/statistics/statisticsui.ui
@@ -17,6 +15,5 @@ if(RUBY_FOUND)
         examples/statistics/statistics.rb 
         DESTINATION ${DATA_INSTALL_DIR}/klinkstatus/scripts)
         
-endif(RUBY_FOUND)
 
 
